addon/components/frost-button.js

```diff
@@ -227,12 +246,17 @@ export default Component.extend(PropTypeMixin, {
     }
   }),
 
+  // FIXME: jsdoc
```

Comment:
I'm not sure I understand your proposal. 

1. Are you saying that not all methods should have JSDoc comments? Which ones should and which ones shouldn't? Is it a judgement call? If so, whose judgement? I've found myself often times looking to work on a function that someone else wrote and not understanding what the parameters are supposed to represent, it takes me quite a while to track down what they were trying to do, particularly if that person isn't available to discuss it. 

2. That is precisely why we have linting rules to make sure that additional parameters are reflected in the comment. If you're talking about the description of the method, then yes, that is something that I care a lot less about than the description of the inputs and outputs. (`@param` and `@returns`). 

3. I'm almost never in favor of writing more. I'm lazy, as all engineers should be. Lazy people automate things. The way we've addressed this in the past is by signing off that an obvious method doesn't require a "proper" comment, but `/* obvious */` can do. That distinguishes between someone consciously deciding to flag a method as obvious vs. just forgetting to add documentation to the method. Then in review, the merits of whether the method should qualify for `/* obvious */` treatment can be discussed on a case-by-case basis. 

4. I'm not sure how having comments gets anyone off the hook for naming a variable properly. That comes down to code review though. I don't think there's any automated system that can do that for us, and I don't think documenting a variable lowers the need for naming it properly to begin with. As long as we are in agreement to that statement, then I don't think documenting the variable hurts. People still need to review the code and sign off on it. Whether there are comments or not, that check still needs to happen. 

5. I don't buy this one. If someone is new to any domain, there are things to learn. I'd rank learning how to format 
```js
/**
 * Description
 * @param {type} name - description
 * @returns {type} description
 */
```
below the bottom of anything else I've had to learn when writing javascript, particularly Ember. Plus, we have lint rules that tell you when it's not working. True, they can probably have better error messages, but saying that it's too much of a strain to have developers learn JSDoc is not what I'd consider a valid argument against it.